docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -65,18 +76,16 @@ The following example shows how to create a gallery-like flexbox layout using th
     toolBar.JustifyContent = FlexContainer.Justification.JustifySpaceBetween;
     ```
 
-5.  Use the toolbar's `Flex` property to make the toolbar and the main content share the height of the main container, so that the toolbar occupies 10 percent of the vertical space and the content occupies the remainder:
+6.  Use `Flex` property of the toolbar to specify the area to be occupied within the main container. The height main container is shared between toolbar and content as specified in next step. Since the main container displays items vertically and the toolbar `Flex` property is set to 0.1, the toolbar will occupy 10% of vertical and 100% of horizontal available space. The remaining area is intended for the content:
```

Comment:
Slight modification needed ( you had missed **of**) from previous comment 

```suggestion
6.  Use `Flex` property of the toolbar to specify the area to be occupied within the main container. The height of the main container is shared between the toolbar and the content as specified in the next step. Since the main container displays items vertically and the toolbar `Flex` property is set to 0.1, the toolbar will occupy 10% of vertical and 100% of horizontal available space. The remaining area is intended for the content:

Reply:
One opinion to "as specified in the next step"

The original "added in step 7" was decorating "content". So the meaning was "the content which is added in step 7"
However it is changed to "as specified in the next step", so it becomes to decorate "The height of the main container is shared between the toolbar and the content".

The meaning is changed, and even more, in step 7, "creating a content"is described rather than "sharing the height of the main container between the toolbar and the content". "Occupy 90 percent of available space on the cross axis" is kind of description of the content property, instead of focusing "sharing the height".

So I think we'd batter
1. to remove "as specified in the next step", as it is wrong and redundant, or
2. change  "content as specified in the next step" to other, for example like "content that will be created", if you want to keep.

In my thought removing is better, but I'd like to get your opinion. Would you please let me know your thought?

Reply:
Sure, @mijin85cho Although, literally both mean the same though not completely, 

added in step 7 

as specified in the next step

 The height of the main container is shared between the toolbar and the content as added in (step 7). 
 The height of the main container is shared between the toolbar and the content as specified in the next step.

But yes your suggestion is good, but may not be relevant, as step 6 is talking about specifically the height/position/ location, thats where step 7 gets reffered to. 

@pkalota could you please provide your valuable opinion on this.

Reply:
Maybe it is better to firstly create toolbar and content and after that focus on flex property? It is more clear to put `Flex` properties of both items next to them. It shows connection between `Flex` properties of these items and it will be easier for user to understand this dependency. @mijin85cho and @nawab-samsung please take a look at below change and let me know what are you thinking about that.


6.  To display the image, create the content area by adding a third flex container at the bottom of the main container. Show the image in the center of the content area container by centering the item horizontally and vertically inside the area:

    ```csharp
    /// Create the content area
    FlexContainer content = new FlexContainer();
    content.ParentOrigin = ParentOrigin.TopLeft;
    content.PivotPoint = PivotPoint.TopLeft;
    /// Place items horizontally
    content.FlexDirection = FlexContainer.FlexDirectionType.Row;
    /// Align items horizontally center
    content.JustifyContent = FlexContainer.Justification.JustifyCenter;
    /// Align items vertically center
    content.AlignItems = FlexContainer.Alignment.AlignCenter;
    /// Add the content area to the main container
    flexContainer.Add(content);
    ```

7.  Use `Flex` property of the `FlexContainer` class to specify the area to be occupied within the main container. Since the main container displays items vertically its height is shared between the toolbar and the content. Setting `toolbar.Flex` to 0.1 and `content.Flex` to 0.9 means that `toolbar` will occupy 10% of vertical space and `content` will occupy 90% of vertical space of the main container. Width of items will be eqaul to width of main container.

    ```csharp
    /// Occupy 10 percent of available space on the cross axis
    toolBar.Flex = 0.1f;
    /// Occupy 90 percent of available space on the cross axis
    content.Flex = 0.9f;
    ``` 

If you prefer previous version, I think 2nd @mijin85cho's option 

>change "content as specified in the next step" to other, for example like "content that will be created", if you want to keep."

is the best one here. Thanks that we are avoiding user's question "What is this content?".